libnavigation-core/src/main/java/com/mapbox/navigation/core/telemetry/events/NavigationCustomEvent.kt

```diff
@@ -0,0 +1,51 @@
+package com.mapbox.navigation.core.telemetry.events
+
+import android.annotation.SuppressLint
+import android.os.Build
+import android.os.Parcel
+import com.google.gson.Gson
+import com.mapbox.android.telemetry.Event
+import com.mapbox.android.telemetry.TelemetryUtils
+import com.mapbox.navigation.base.metrics.MetricEvent
+import com.mapbox.navigation.base.metrics.NavigationMetrics
+
+@SuppressLint("ParcelCreator")
+internal class NavigationCustomEvent : Event(), MetricEvent {
+
+    private companion object {
+        private val OPERATING_SYSTEM = "Android - ${Build.VERSION.RELEASE}"
+    }
+
+    var type: String = ""
+    var payload: String? = null
+    val version: String = "2.4"
```

Comment:
Is that a `DropIn UI` version? Shouldn't that come from `build.gradle` via `BuildConfig`?

Reply:
This is the [required schema](https://github.com/mapbox/event-schema/blob/master/lib/base-schemas/navigation.customEvent.js) and the value should be hardcoded to `2.4` as per the schema.